seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/dal/entity/ScriptParam.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.app.dal.entity;
+
+import java.util.Date;
+
+public class ScriptParam {
+    private Integer id;
+
+    private Integer scriptId;
+
+    private String key;
+
+    private String value;
+
+    private Byte status;
```

Comment:
I think use` int` is enough.
```suggestion
    private Integer status;
```

Reply:
`Byte` takes up less database disk

Reply:
I didn't see your table schema, in my knowledge, we can use tinyint for this field, tinyint only need 1 byte.

Reply:
> I didn't see your table schema, in my knowledge, we can use tinyint for this field, tinyint only need 1 byte.

Yes, I'm using tinyInt, which in JAVA type is Byte.

I will show ddl after all entity's api been completed.